magma/magma
# [CI][Jenkins] Pipeline Improvements

## Summary
Here's few Jenkins pipeline Updates:
 - integration with Slack by @mattymo 
 - refactoring of the S11 Test stages (exclude TF server + add test VM reload )
## Test Plan
- Run Jenkins Pipepline
- See messages in the CI slack channel
- See No "S11 Tests stuck issues"

## Additional Information
 - fix for  [issue #3083 ]
 - [skipJenkins]
